frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.unit.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ORDERS, SAMPLE_DATABASE } from "__support__/sample_database_fixture";
+import StructuredQuery from "metabase-lib/queries/StructuredQuery";
+import ExpressionWidget, { ExpressionWidgetProps } from "./ExpressionWidget";
+
+describe("ExpressionWidget", () => {
+  it("should render proper controls", () => {
+    setup();
+
+    expect(screen.getByText("Expression")).toBeInTheDocument();
+    expect(screen.getByText("Name")).toBeInTheDocument();
+
+    expect(screen.getByText("Cancel")).toBeInTheDocument();
+    expect(screen.getByText("Done")).toBeInTheDocument();
+  });
+
+  it("should render help icon with tooltip", () => {
+    setup();
+
+    const iconEl = screen.getByRole("img");
+    expect(iconEl).toBeInTheDocument();
+    expect(iconEl).toHaveClass("Icon-info");
```

Comment:
We can use a `getIcon` helper from `__support__/ui` to avoid the CSS class assertion:

```jsx
import { getIcon } from "__support__/ui"

const icon = getIcon("info")
expect(icon).toBeInTheDocument()
userEvent.hover(icon)
```

Reply:
👆 note that the path has changed since - https://github.com/metabase/metabase/pull/28749